doc/developer/alembic_on_custom_schema.rst

```diff
@@ -0,0 +1,103 @@
+.. _alembic_on_custom_tables:
+
+Alembic on custom tables
+========================
+
+If you want to manage custom tables (GMF tables; not in the ``main`` nor the ``static`` schema)
+with an alembic you must setup a custom alembic process. Here's an example. Others solutions
+are possible, but do not try to reuse the existing ``alembic.ini``.
```

Comment:
Can we say why not?

Reply:
because you will not be able to specify a schema through the configuration levels.